src/openforms/scss/components/admin/_logic-action.scss

```diff
@@ -16,6 +25,17 @@ $logic-line-breakpoint: 1800px !default;
     width: calc(100% - 1em);
   }
 
+  &__row--has-errors {
+
+    .logic-action__actions {
+      padding-top: $error-height;
+    }
+
+    .dsl-editor__node:not(.errors) {
+      padding-top: $error-height;
+    }
```

Comment:
I doubt this works if there are multiple validation errors for the same node, but it could be a good first approach.